osf/migrations/0213_registrationschema_providers.py

```diff
@@ -8,7 +8,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('osf', '0211_auto_20200709_1320'),
+        ('osf', '0212_auto_20200710_1734'),
```

Comment:
Or it may be that this is out of order. This should be 0212 and the migration about should be 0213, since this migration is already on develop. But if we do it this way, we're going to break things with migrations I would imagine. Sorry if I put my comment in the wrong migration before.

Reply:
I merged these two migrations together into one auto-generated migration.

Reply:
I don't think that's going to work (see https://github.com/CenterForOpenScience/osf.io/pull/9435#commitcomment-40971145 )

Reply:
Ok I reverted it, should be g2g now.